cmd/import_data.go

```diff
@@ -18,13 +18,16 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/GoogleCloudPlatform/spanner-migration-tool/conversion"
```

Comment:
this command does schema and data. Can you use a more indicative name ?

Reply:
Done.